### ISSUE TITLE
New vulnerability 

### ISSUE THREAD

**Issue by opravil-jan** [mapbox/node-pre-gyp#633]
Hi, 
in package node-fetch is vulnerability. Can you fix it please?
```

node-fetch  <2.6.7
Severity: high
node-fetch is vulnerable to Exposure of Sensitive Information to an Unauthorized Actor - https://github.com/advisories/GHSA-r683-j2x4-v87g

```
Thanks

**Comment by cjwilsontech** [comment]
Created a PR to fix. https://github.com/mapbox/node-pre-gyp/pull/634

**Comment by axrj** [comment]
Hi, this is fixed in v1.0.9.
